internal/metrics/collectors/processes.go

```diff
@@ -57,7 +57,7 @@ func getWorkerProcesses() (int, int, error) {
 		}
 
 		cmdlineFile := fmt.Sprintf("/proc/%v/cmdline", folder.Name())
-		content, err := ioutil.ReadFile(cmdlineFile)
+		content, err := os.ReadFile(cmdlineFile)
```

Comment:
Looks like this introducing a linting error (for some reason it's only flagged with the switch from ioutil -> os) - it can be mitigated by doing something like:
```
<...>
cmdlineFile := fmt.Sprintf("/proc/%v/cmdline", folder.Name())
cmdlineFile = filepath.Clean(cmdlineFile)
if !strings.HasPrefix(cmdlineFile, "/proc/") {
	panic(fmt.Errorf("unsafe input"))
}
content, err := os.ReadFile(cmdlineFile)
<...>
```
See https://securego.io/docs/rules/g304.html

Reply:
Fixed in 375732b4621574414ee50ffa5f11d6beb5fd5ae4. Please take a look again. Thanks.

Reply:
`make lint` is not returning the error anymore after the latest ocmmit, but the CI still failed (?)

Reply:
@Juneezee sorry about that, this should fix the discrepancies https://github.com/nginxinc/kubernetes-ingress/pull/2365